src/modules/notebook-panel/NotebookPanel.js

```diff
@@ -19,6 +19,7 @@ import MeasurementDetailPage from '../measurements/MeasurementDetail';
 import MeasurementsPage from '../measurements/Measurements';
 import Nesting from '../nesting/Nesting';
 import NotesPage from '../notes/Notes';
+import OtherFeaturesPage from '../petrology/OtherFeaturesPage';
```

Comment:
Other Features is not a part of petrology. It should be in it's own folder called `other-features` just like in SS1.